inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/fetcher/dtos/JobProfileDto.java

```diff
@@ -36,7 +36,7 @@
     public static final String DEFAULT_CHANNEL = "org.apache.inlong.agent.plugin.channel.MemoryChannel";
     public static final String MANAGER_JOB = "MANAGER_JOB";
     public static final String DEFAULT_DATAPROXY_SINK = "org.apache.inlong.agent.plugin.sinks.ProxySink";
-    public static final String DEFAULT_SOURCE = "org.apache.inlong.agent.plugin.sources.TextFileSource";
+    public static final String DEFAULT_SOURCE = "org.apache.inlong.agent.plugin.sources.DataBaseSource";
```

Comment:
DatabaseSqlSource could be better, since there will be DatabaseBinlog source

Reply:
get